src/main/java/jpo/sdw/depositor/DepositorProperties.java

```diff
@@ -20,7 +20,7 @@ public class DepositorProperties implements EnvironmentAware {
 
    private static final String DEFAULT_GROUP_ID = "usdot.jpo.sdw";
    private static final String DEFAULT_KAFKA_PORT = "9092";
-   private static final String DEFAULT_DESTINATION_URL = "https://sdx-service.trihydro.com/api/deposit";
+   private static final String DEFAULT_DESTINATION_URL = "https://sdx-service.trihydro.com/api/deposit-multi";
```

Comment:
How did this destination url get updated from env? I leave the env values empty and see this default destination url is not used.
![image](https://github.com/usdot-jpo-ode/jpo-sdw-depositor/assets/62157949/4decd402-a1bc-414d-b675-5c52cf26a126)

The subscription topics are set to default correctly.
![image](https://github.com/usdot-jpo-ode/jpo-sdw-depositor/assets/62157949/4f69deea-ed7d-49aa-b528-6cc5341ef066)

Reply:
An empty string check has been added to the DepositorProperties class. In the case where the SDW_DESTINATION_URL environment variable is commented out or missing, the default URL will now be used. Thanks for catching this!